DEVELOPMENT.md

```diff
@@ -171,28 +171,52 @@ To generate a production GAPIC API:
 2. Point to local gapic-generator-java
 
    Normally, googleapis's build pulls in googleapis/gapic-generator-java from the
-   Internet:
-
-   ```
-   # Java microgenerator.
-   …
-   _gapic_generator_java_version = "2.1.0"
+   Internet, we need to build a local SNAPSHOT jar first and point googleapis to the SNAPSHOT jar and local copy of the repo.
```

Comment:
Let me know if you have any issues with the getting local development set up. I was able to get this set up by following what Blake wrote in this PR.

Reply:
`mvn fmt: check` and `mvn fmt:format` do not work from root for me:
```
[ERROR] No plugin found for prefix 'fmt' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/alicejli/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
```

Not sure if that was part of a separate fix, but wanted to highlight just in case.

Otherwise the steps work with the addition of installing Bazelisk (https://github.com/bazelbuild/bazelisk) per @lqiu96's advice. If we can add that bit in as well that would be great. Thank you!

Reply:
Ah, thanks for running the docs. I think this is coming from the pre-commit hooks which is where I've also come across. I've ended up disabling them for now (due to the monorepo migration which I think have caused them to fail). I'll add a quick note in the file about this and whenever someone has capacity to fix them, they can also update this file.